README.md

```diff
@@ -233,8 +339,8 @@ aws.secret.access.key=YOUR_AWS_SECRET_ACCESS_KEY
 #AWS Region
 aws.s3.region=us-east-1
 
-#S3 prefix template
-aws.s3.prefix=your-prefix/{{topic}}/
+#File name template
+file.name.template=your-prefix/{{topic}}
```

Comment:
This example is technically valid, but not very useful (not something any user would like to have). I propose to re-use this one from the GCS connector:
```suggestion
file.name.template=dir1/dir2/{{topic}}-{{partition}}-{{start_offset:padding=true}}.gz
```